README.md

```diff
@@ -110,76 +109,35 @@ npm run test:watch
-
-- I will receive a second email the day before the event to remind me.
-
-- After the event, I will automatically get emails notifying me of subsequent events.
-
-- I can filter all events in my location by tag/interests.
-
-### As an organizer
-
-- I can create a chapter.
-
-- I can edit details about the chapter, including a Slack/Discord/Facebook/WeChat/WhatsApp link participants can join to discuss and coordinate events.
-
-- I can create events, and set their location and capacity.
-
-- I can cancel events.
-
-- I can email the entire list of participants.
-
-- I can ban a participant whom I believe is toxic or who has previously broken my organization's code of conduct.
-
-- I can add a venue sponsor to the event with a link to their website as a way of thanking them for hosting.
-
-- I can add a food sponsor to the event with a link to their website as a way of thanking them for food.
-
-- I can see how many times a participant has come to the event as well as their attendance rate.
-
-- I can check-in attendees on the event registration desk with their `email_id` or `chapter_id`.
+1. Please check your idea does not already exist as an [MVP issue](https://github.com/freeCodeCamp/chapter/labels/MVP), or in the [post-MVP list](https://github.com/freeCodeCamp/chapter/issues/84).
+1. Create a new "Issue" and post details about the suggested user story.
 
 ## Roadmap
 
-1. Design the schema.
-2. Set up the API endpoints.
-3. Build the web client and let other developers use the API to build mobile clients and voice interface clients.
+The on-going [project Roadmap conversation](https://github.com/freeCodeCamp/chapter/issues/47) is regularly updated to reflect the overall progress and for higher-level discussions.
 
-Quincy Larson is the project lead. [FreeCodeCamp](https://www.freecodecamp.org) will start "dogfooding" this as soon as possible with several of its local study groups.
+Quincy Larson is the project lead. [FreeCodeCamp](https://www.freecodecamp.org) will start "dogfooding" the MVP with several of its local study groups.
 
 Here's an out-dated example of an app with similar functionality: [The freeCodeCamp Study Group Directory](https://study-group-directory.freecodecamp.org).
 
-You should [join our Discord server](https://discord.gg/vbRUYWS) to get connected with people interested in this project and to be aware of our future announcements.
 
 ## Contributing
 
-[**Take part in discussions or contribute code to this opensource codebase.**](CONTRIBUTING.md)
+* You should [join our Discord server](https://discord.gg/vbRUYWS) to get connected with people interested in this project and to be aware of our future announcements.
+* Please read the [**suggested steps to contribute code to the Chapter project**](CONTRIBUTING.md) below forking or submitting any pull requests.
```

Comment:
```suggestion
* Please read the [**suggested steps to contribute code to the Chapter project**](CONTRIBUTING.md) before forking or submitting any pull requests.
```

Reply:
Thanks Scott for the typo fix. @vkWeb had suggested not having this new bullet.  I don't think it's critical and perhaps it's better in the CONTRIBUTING.md.

The purpose was to more to encourage new user story suggestions to create a new issue instead of dropping new ideas in comments, where it's likely to get lose.

The bullet about checking that the idea doesn't already exist was secondary, but still a reminder for anybody who might not know where to look for the full set of existing suggestions.